cscs-checks/apps/jupyter/src/tf-hvd-sgd-ipc-tf2.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
+import ipcmagic
```

Comment:
If this is not used, add a `# noqa: F401` annotation comment in this line so as to be ignored by flake.